packages/gantt/frontend/sirius-components-gantt/src/representation/Gantt.tsx

```diff
@@ -10,20 +10,28 @@
  * Contributors:
  *     Obeo - initial API and implementation
  *******************************************************************************/
-import {
-  Gantt as GanttDiagram,
-  Task,
-  TaskListColumn,
-  TaskListColumnEnum,
-  ViewMode,
-} from '@ObeoNetwork/gantt-task-react';
+import '@ObeoNetwork/gantt-task-react';
+import { ColorStyles, Column, Gantt as GanttDiagram, Task, TaskOrEmpty, ViewMode } from '@ObeoNetwork/gantt-task-react';
 import '@ObeoNetwork/gantt-task-react/dist/style.css';
 import { Selection } from '@eclipse-sirius/sirius-components-core';
-import { useState } from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import { useRef, useState } from 'react';
+import { DndProvider } from 'react-dnd';
+import { HTML5Backend } from 'react-dnd-html5-backend';
```

Comment:
Remaining trace of react-dnd
It should be deleted since gantt-task-react don't use it anymore

Reply:
oups. yes of course